DOCUMENTATION.md

```diff
@@ -357,6 +358,28 @@ assets you have access to. 
 content-cli list assets -p <your-chosen-profile> --assetType SCENARIO
 ```
 
+### List assignments
+
+With this command you can retrieve a list of possible variable assignment values for a variable type.
+The command takes your permissions into consideration and only lists the values you have access to.  
+
+-   It is also possible to download packages in JSON format by adding '--json' option.
+
+```
+content-cli list assignments -p <your-chosen-profile> --variableType <your-chosen-variable-type>
+```
+
+Currently, only variables of type DATA_MODEL and CONNECTION are supported.  
+
+The command includes an optional --params option for additional value filtering. 
+Parameters should be provided in CSV format. 
+For instance, when dealing with variables of type CONNECTION, you can use the --params option to retrieve 
+only the potential assignment values for connections with the appName 'Celonis' using the following command:
+
+```
+content-cli list assignments -p <your-chosen-profile> --variableType CONNECTION --params appName=Celonis
```

Comment:
This part looks good to me.

The previous PR which updated the [Batch Import packages into Studio](https://github.com/celonis/content-cli/blob/master/DOCUMENTATION.md#batch-import-packages-into-studio) I think it's no that clear.

We are saying that by default the variables are assigned, but that is not true. I think the wording should be that if the users have assigned the variables in the manifest file, then those values will be saved.

Also for assigning only the data models, I think we should mention that that is deprecated and will be removed in the future releases. What do you think?

Reply:
Regarding the wording of the variable assignments, I think the second part of the sentence you mentioned clarifies the point you are making: `... as defined in the manifest.yml file of the exported packages.`.

As for the second point I agree, and added it to the doc.